tools/ir-generator/ir-generator.ypp

```diff
@@ -44,6 +44,7 @@ limitations under the License.
 
 static void yyerror(const char *fmt, ...);
 static std::vector<IrElement*> global;
+CommentBlock *currCommentBlock = nullptr;
```

Comment:
This should be static, to make clear that it is only accessed in this source file